ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -55,7 +55,8 @@ export const EntriesList: React.FC<EntriesListProps> = ({
   const [startTime, setStartTime] = useState(0);
   const [truncatedTimestamp, setTruncatedTimestamp] = useState(0);
 
-  const leftOffBottom = entries.length > 0 ? entries[entries.length - 1].id + 1 : -1;
+  const zeroPad = (num, places) => String(num).padStart(places, '0')
+  const leftOffBottom = entries.length > 0 ? zeroPad(+entries[entries.length - 1].id + 1) : "";
```

Comment:
Could you please revert this change? We need to understand caused `-1` in `develop` first. The front-end code shouldn't alter the index values.

Reply:
sure I will revert it..

Reply:
I did it with this commit https://github.com/up9inc/mizu/pull/984/commits/e1ba1cefbde0d11b4c2f3639e5513bab0fde814b I hope this is what you mean